cmd/support-perf.go

```diff
@@ -99,6 +99,8 @@ var supportPerfCmd = cli.Command{
 USAGE:
   {{.HelpName}} [COMMAND] [FLAGS] TARGET
 
+COMMAND:
+  valid values are 'drive', 'net', 'object' and 'site-replication'
```

Comment:
these are not documented on purpose @shtripat

Reply:
ok, closing the PR for now